src/liboslexec/wide/wide_opdictionary.cpp

```diff
@@ -169,7 +169,7 @@ template<typename ValueT> struct DictValueGetter<ValueT, false> {
 
             ValueT value;
             int result = context->dict_value(nodeID, attribname, wdest.type(),
-                                             &value);
+                                             &value, false);
```

Comment:
Add comment on `false `argument identifying the name of parameter `/* treat_ustrings_as_hash */`